Snippets/backstat.sqlpromptsnippet

```diff
@@ -0,0 +1,13 @@
+    SELECT  r.session_id as spid, command, start_time, percent_complete,
```

Comment:
Looks helpful, but this isn't in the usual .sqlpromptsnippet style.

A question - why it is called backstat? Is that short for backup statistics?

Also a few suggestions on the SQL itself before merging:

1. I'd replace the verbose CASE WHEN with the more succinct IIF(condition, iftrue, if false)
2. Capitalise GETDATE() / VARCHAR / DATEADD
3. Capitalise VARCHAR
4. Don't need the extra brackets on line 13 around the WHERE clause
5. I'd drop the logic that adds the (s) on the end of hour, min, sec - nice touch but feels a little overkill to me (the cost of keeping it being that the code is that much longer to read for a newbie)